### ISSUE TITLE
Presence of guard-minitest development dependency breaks Travis build for Ruby 1.9

### ISSUE THREAD

**Issue by mattbrictson** [mattbrictson/airbrussh#28]
It looks like when minitest starts, it scans for minitest plugin gems and tries to load `guard-minitest`, which is not compatible with Ruby 1.9.

https://travis-ci.org/mattbrictson/airbrussh/jobs/68626916#L164

```
/home/travis/.rvm/gems/ruby-1.9.3-p551/gems/listen-3.0.0/lib/listen.rb:2:in `require': /home/travis/.rvm/gems/ruby-1.9.3-p551/gems/listen-3.0.0/lib/listen/logger.rb:26: unknown type of %string (SyntaxError)
    %i(fatal error warn info debug).each do |meth|
```

**Comment by mattbrictson** [comment]
Fixed in bc3aa00

**Comment by robd** [comment]
Yes! I had the same fix, but I wanted to see if it was just something local to me first. Thanks for sorting this.
